ls1intum/Artemis
# [Bugfix] - Improve quiz statistics

### Checklist

### Motivation and Context


We have two problems with the drag & drop quiz statistics:

1. The percentages in the bars are not rounded correctly
2. The blank locations in the uploaded picture are not located correctly - For different sizes of the screen, the rectangles cover different parts of the picture (see videos)

### Description

We added a rounding for the bars and adapted the white rectangles a bit so they align with the initially positioned rectangles

### Steps for Testing


1. Log in to Artemis
2. Create a drag & drop quiz
3. Place the white rectangles (the places where the students are supposted to drop a solution onto) on a place where you can easily identify if the rectangles are places wrongly afterwards
4. Conduct the quiz
5. Look into the statistics and navigate into the D&D quiz question (on the bottom left, dropdown menu, then select the question)
6. Make sure the percentages above the bars are rounded correctly
7. Resize the window and make sure the white rectangles stay at the right place all the time
### Review Progress





- Code Review
- Manual Tests

### Screenshots



Example before:

https://user-images.githubusercontent.com/44401560/131211830-949c7943-90dd-4397-92cd-7ce3980b107c.mov


Example after:

https://user-images.githubusercontent.com/44401560/131211811-e2947193-81fb-4269-b3c8-051bf2e93ebb.mov
